ablator/analysis/results.py

```diff
@@ -39,23 +39,23 @@ def read_result(config_type: type[ConfigBase], json_path: Path) -> pd.DataFrame
     Examples
     --------
     >>> result json file:
+    [{
+    "train_loss": 10.35,
+    "val_loss": NaN,
+    "current_epoch": 1,
+    },
     {
-    "run_id": "run_1",
-    "accuracy": 0.85,
-    "loss": 0.35
-    }
-    {
-    "run_id": "run_2",
-    "accuracy": 0.87,
-    "loss": 0.32
-    }
+    "train_loss": 3.89,
+    "val_loss": 7.04,
+    "current_epoch": 2,
+    }]
     >>> config file
-    experiment_name: "My Experiment"
-    batch_size: 64
+    experiment_dir: "C:\tmp\results\experiment_8925_9991"
```

Comment:
Can we switch to unix based path e.g. Ubuntu

Reply:
Sure